### ISSUE TITLE
fga store create - breaking change v0.5

### ISSUE THREAD

**Issue by atoys-srsa** [openfga/cli#354]
We currently parse the response from `fga store create --model Model.fga` with `jq` to set environment variables with store and model ids as per this documented example: https://github.com/openfga/cli?tab=readme-ov-file#response-1

I'm not sure if it is the progress bar itself or the additional `✅ Store imported` line before the json, but this has broken our CICD process. Full response:

```
✅ Store imported                                                                                                                                                                                                                                                                                
{
  "store": {
    "created_at":"2024-06-24T20:24:54.656783Z",
    "id":"01J15YRJ.....WKRHRC957",
    "name":"some_name",
    "updated_at":"2024-06-24T20:24:54.656783Z"
  },
  "model": {
    "authorization_model_id":"01J15YR.......EFSJN4TD03"
  }
}
```

Error:
`jq: parse error: Invalid numeric literal at line 1, column 11`

**Comment by rhamzeh** [comment]
Thanks for raising this @atoys-srsa - we're considering defaulting to the pretty output and adding a flag, such as `--output-format=json` or similar for this use-case.

Would that work for you?

**Comment by aaguiarz** [comment]
@rhamzeh for `store create` we should not be writing "Store Imported", right?

**Comment by Siddhant-K-code** [comment]
ahh!! Change was introduced in https://github.com/openfga/cli/pull/348 by me 🙈 --- Sorry for breaking your flow 🙏🏼

We can remove `✅ Store imported` ([code ref](https://github.com/Siddhant-K-code/openfga-cli/blob/main/cmd/store/import.go#L186)) and just show the Progress bar while it's in progress, then just show the output of the store which got created. Wdyt?

**Comment by ewanharris** [comment]
@atoys-srsa, we just released [v0.5.1](https://github.com/openfga/cli/releases/tag/v0.5.1) that includes a fix for this issue. Please let us know if you face any further issues

**Comment by atoys-srsa** [comment]
@ewanharris Thanks for the quick response! Confirmed working.
